backend/backend/.env.prod

```diff
@@ -0,0 +1,19 @@
+DJANGO_SECRET_KEY=t38x6_9$&3cp!6v-t6tmhak%mh=g52^iw%e2o_%(4d5=d9!v4n
+DJANGO_ALLOWED_HOSTS=localhost 127.0.0.1 [::1]
+DEBUG=False
+
+PORTAIL_HOSTNAME=PORTAIL_HOSTNAME
+
+DATABASE_ENGINE=django.db.backends.postgresql
+DATABASE_NAME=portail_prod
+DATABASE_USER=root
+DATABASE_PASSWORD=password
+DATABASE_HOST=db
+DATABASE_PORT=5432
+
+BACKEND_HOST=www.portail-des-eleves-v2-api.com
+CORS_ORIGIN_WHITELIST=localhost:5000
+
+# Should begin with `-----BEGIN PUBLIC KEY-----` and end with `-----END PUBLIC KEY-----`.
+# Please pay attention to the new lines. They should be encoded as `\n` in this file.
+JWT_PUBLIC_KEY=-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAoWPv5B06KeQPru1unPs1\nk/iAME1nIEsoWJIkmaP5broZcUpztOpmRAf8+RCk5oAXQrJ87SoIs8UG4fKCduNa\nFXsKvv1zn6z+S92ScpHOYmuxO9mlL1np9NbhRdiSDwIomdpXNZ+ttF+Ll0nO87wU\naKvTdwLhLswSLEjt2YwmKOhgpSPe2mLkk5Me1v7iAfj4ermisaV/WLu7QbelboZO\nYp3yhOkPVKhvJU4fBkx/cnkcvLMo8dx5sM3TQonZB5oiv1w4YWz7nBjbmH3bv+9u\nAiLpr9K2ueieZbM1R3f7vhS44pNUI5LYRYA4NHEv1O+n5bE8BuvnVMG5qGP6fB8J\nuwIDAQAB\n-----END PUBLIC KEY-----
```

Comment:
The point of a .env file is (among other) not to publish secrets in git. Please remove `.env.prod` and replace it with `.env.prod.dist` containing dummy values adding comments to explain of to fill the real values

Reply:
Not resolved yet